tests/test_structures/test_datasample.py

```diff
@@ -156,3 +157,20 @@ def test_set_pred_score(self):
         data_sample = ClsDataSample()
         data_sample.set_pred_score(torch.tensor([0.1, 0.1, 0.6, 0.1, 0.1]))
         self.assertEqual(data_sample.pred_label.num_classes, 5)
+
+
+class TestMultiTaskDataSample(GenericTest):
+    def test_set_gt_label(self):
+        self._test_set_label(key='gt_label',data_sample=MultiTaskDataSample().to_cls_data_samples())
+
+    def test_set_pred_label(self):
+        self._test_set_label(key='pred_label',data_sample=MultiTaskDataSample().to_cls_data_samples())
+
+    def test_get_task_mask(self):
+        data_samples = []
+        gt_label = {}
+        gt_label['task1'] = 1
+        data_sample = MultiTaskDataSample(['task1']).to_cls_data_samples().set_gt_label(gt_label)
+        data_samples.append(data_sample)
+        self.assertTrue(data_samples.get_task_mask('task1'),True)
+        self.assertFalse(data_samples.get_task_mask('task2'),False)
```

Comment:
Let's test first without any tasks as input like `MultiTaskDataSample()`

we could add the tasks optionnaly `MultiTaskDataSample(metainfo={
  'task0': {num_classes='10'},
  'task1': {}
})`


```suggestion
        data_sample = MultiTaskDataSample().set_gt_label(gt_label)
        self.assertTrue(data_sample.get_task_mask('task1'),True)
        self.assertTrue(data_sample.get_task_mask('task2'),False)
```